docs/src/Accessibility.doc.js

```diff
@@ -0,0 +1,250 @@
+
+        Another key aspect to keyboard navigation is [focus management](https://css-tricks.com/focus-management-and-inert/). Users should never lose their place within a page, and their current point of focus should always be clear visually. Some components, like Modals or Popovers, act as focus traps, meaning the user's focus should only rotate between items inside the Modal or Popover, and anything below these items is not reachable by keyboard. When a component like Modal or Popover is dismissed or closed, the user's focus should go back to the item that triggered the component.
+    `}
+    />
+    <MainSection.Subsection
+      columns={2}
+      title="Available hooks"
+      description={`Gestalt provides two functions that help create more accessible experiences: \`useFocusVisible\` and \`useReducedMotion\`.`}
+    >
+      <MainSection.Card
+        cardSize="md"
+        title="useFocusVisible"
+        description={`
+          \`useFocusVisible\` manages focus interactions on the page and determines whether a focus ring should be shown. If a user interacts with a mouse/touch, then the focus is not visible. When the user interacts with the keyboard, then the focus is visible.
+
+          References:
+          <ul>
+            <li><a href="https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html">WCAG 2.4.7: Focus Visible</a></li>
+            <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible">:focus-visible CSS pseudo-class</a></li>
+          </ul>
+        `}
+        defaultCode={`
+function Example() {
+  const { isFocusVisible } = useFocusVisible();
+  const [ focussedButton1, setFocussedButton1 ] = React.useState(false);
+  const [ focussedButton2, setFocussedButton2 ] = React.useState(false);
+
+  return (
+    <Flex alignItems="start" direction="column" gap={4}>
+      <Flex alignItems="center" gap={4}>
+        <Text>With focus visible</Text>
+        <button
+          onBlur={() => setFocussedButton1(false)}
+          onFocus={() => setFocussedButton1(true)}
+          style={{
+            outline: 'none',
+            boxShadow: isFocusVisible && focussedButton1 ? "0 0 0 4px rgba(0, 132, 255, 0.5)" : null
+          }}
+        >
+          <Text>Button 1</Text>
+        </button>
+      </Flex>
+      <Flex alignItems="center" gap={4}>
+        <Text>Without focus visible</Text>
+        <button
+          onBlur={() => setFocussedButton2(false)}
+          onFocus={() => setFocussedButton2(true)}
+          style={{
+            outline: 'none',
+            boxShadow: focussedButton2 ? "0 0 0 4px rgba(0, 132, 255, 0.5)" : null
+          }}
+        >
+          <Text>Button 2</Text>
+        </button>
+      </Flex>
+    </Flex>
+  );
+}`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="useReducedMotion"
+        description={`
+          \`useReducedMotion\` allows a user to request that the system minimize the amount of non-essential motion.
```

Comment:
I think this should perhaps be more direct and say that all animations should first use this hook, and then show a non-animated version if requested? What do you think?

Reply:
I'm going to save this for when we take a full pass at the stand alone page, but yeah, we can probably update this part